Document/0x06b-Basic-Security-Testing.md

```diff
@@ -343,60 +350,88 @@ $ ssh -p 2222 root@localhost
+- `/private/var/mobile/Containers/Data/Application/CC24A101-A668-4F77-B410-2FF47A281D05` contains temporary files which aren't needed between application launches.
 
-The Application subdirectory, which is inside the Bundle subdirectory, contains the name of the app. The static installer files are in the Application directory, and all user data is in the Data directory.
+The following illustration represents the application folder structure:
 
-The random string in the URI is the application's GUID. Every app installation has a unique GUID. There is no relationship between an app's Bundle GUID and its Data GUID.
+<img src="Images/Chapters/0x06a/iOS_Folder_Structure.png" alt="iOS App Folder Structure" width="350">
 
 ##### Copying App Data Files via SSH and SCP
 
-App files are stored in the Data directory. To identify the correct path, SSH into the device and use IPA Installer Console to retrieve the package information (as shown previously):
+As we know now, files from our app are stored in the Data directory. You can now simply archive the Data directory with `tar` and pull it from the device with `scp`:
 
 ```shell
-iPhone:~ root# ipainstaller -l
-...
-sg.vp.UnCrackable1
-
-iPhone:~ root# ipainstaller -i sg.vp.UnCrackable1
-Identifier: sg.vp.UnCrackable1
-Version: 1
-Short Version: 1.0
-Name: UnCrackable1
-Display Name: UnCrackable Level 1
-Bundle: /private/var/mobile/Containers/Bundle/Application/A8BD91A9-3C81-4674-A790-AF8CDCA8A2F1
-Application: /private/var/mobile/Containers/Bundle/Application/A8BD91A9-3C81-4674-A790-AF8CDCA8A2F1/UnCrackable Level 1.app
-Data: /private/var/mobile/Containers/Data/Application/A8AE15EE-DC8B-4F1C-91A5-1FED35258D87
+iPhone:~ root# tar czvf /tmp/data.tgz /private/var/mobile/Containers/Data/Application/CC24A101-A668-4F77-B410-2FF47A281D05
+iPhone:~ root# exit
+$ scp -P 2222 root@localhost:/tmp/data.tgz .
 ```
 
-You can now simply archive the Data directory and pull it from the device with `scp`:
+##### Passionfruit
 
-```shell
-iPhone:~ root# tar czvf /tmp/data.tgz /private/var/mobile/Containers/Data/Application/A8AE15EE-DC8B-4F1C-91A5-1FED35258D87
-iPhone:~ root# exit
-$ scp -P 2222 root@localhost:/tmp/data.tgz .
+After starting Passionfruit you can select the app that is in scope for testing. There are various functions available, of which one is called "Files". When selecting it, you will get a listing of the directories of the app sandbox.
+
+When navigating through the directories and selecting a file, a TextViewer pop-up will show up that illustrates the data either as hex or text. When closing this pop-up you have various options available for the file, including:
+
+- Text viewer
+- SQLite viewer
+- Image viewer
+- Plist viewer
+- Download
+
+<img src="Images/Chapters/0x06b/Passionfruit_files.png" alt="Passiofruit File Options">
+
+##### Objection
+
+When you are starting objection you will find the prompt within the bundle directory.
```

Comment:
we have to be sure to refer to the folders always with upper-/lowercase, e.g. below is Data directory, here bundle, should we put Bundle?